apps/judicial-system/backend/src/app/modules/case/pdf.service.ts

```diff
@@ -78,7 +78,8 @@ export class PdfService {
       ?.filter(
         (caseFile) =>
           caseFile.policeCaseNumber === policeCaseNumber &&
-          caseFile.category === CaseFileCategory.CASE_FILE &&
+          (caseFile.category === CaseFileCategory.CASE_FILE ||
+            caseFile.category === CaseFileCategory.INVOICE) &&
```

Comment:
Invoices should not be included in case files records. They are standalone case files listed under "other case files" in the client.